packages/test-utils/src/jsx.ts

```diff
@@ -69,6 +70,7 @@ export const jsx = createHyperscript({
     htodoli: { type: ELEMENT_TODO_LI },
     htr: { type: ELEMENT_TR },
     hul: { type: ELEMENT_UL },
+    hdefault: { type: ELEMENT_DEFAULT },
```

Comment:
Please define the constant above instead of importing

Reply:
Done!